public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1501 @@
+
+Tarball (gzip)
+~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>.tar.gz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.tar.gz``
+   * - Format description
+     - ``.tar.gz`` is an archive format, often called a tarball, that combines files into a single file and compresses it using ``gzip`` compression.
+   * - Use cases
+     - Our ``.tar.gz`` archives are used to distribute complete file system images along with the kernel for various operating systems and virtualisation platforms. They allow extraction and booting of the entire system on compatible hardware or virtual machines.
+
+Vagrant box
+~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.box``
+   * - Example filename
+     - ``jammy-server-cloudimg-amd64-vagrant.box``
+   * - Format description
+     - The Vagrant box format is used to package and distribute virtual machine environments managed by Vagrant.
+   * - Use cases
+     - These files contain a virtual machine image along with metadata required for Vagrant. Vagrant simplifies the creation and provisioning of virtual environments, making it easier to manage and share development environments across different systems. You can use a ``.box`` file along with a supported provider to quickly set up environments with specific configurations, tools and dependencies. All Vagrant boxes are provider specific, with our boxes having been built for VirtualBox.
+
+Virtual Hard Disk (VHD)
+~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.vhd.tar.gz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-azure.vhd.tar.gz``
+   * - Format description
+     - A VHD is a file format used by virtualisation software to store virtual hard disk images.
+   * - Use cases
+     - 
+       VHD files are used in cloud environments for storing VM disk images. They allow easy deployment and scaling of virtual machines. In general, you can use a ``.vhd`` file to create an instance based on a pre-configured image, ensuring consistency across multiple deployments.
+
+       The VHD files we publish are tailored specifically for use in the Azure cloud. They do not contain standard VM images and will not function outside of Azure, including on-premises Hyper-V or local development environments. 
+
+Other files
+-----------
+This section includes information on checksums, GPG signatures, changelogs and manifest files. These files help verify the integrity and authenticity of the images, provide details on changes between versions and list all included packages.
```

Comment:
It might be nice to point out that the public keys are present
in /usr/share/keyrings/ubuntu-cloudimage-keyring.gpg on ubuntu systems and can be verified using e.g. 
`gpg --verify --keyring /usr/share/keyrings/ubuntu-cloudimage-keyring.gpg SHA256SUMS.gpg SHA256SUMS`
`sha256sum -c SHA256SUMS`

Reply:
Added this as a note.

Reply:
@mstpn Would it be better to move the note to the 'GPG signatures' section?